docs/CONTRIBUTING.md

```diff
@@ -84,6 +84,48 @@ $ make test-json
 ```
 
 
+## Web Interface
+
+If you want to contribute in **source{d} Lookout** [Web Interface](web.md), you should consider the following:
+
+### Dependencies
+
+It is required [Node.js](https://nodejs.org/es) `>=8`; you can check which version of `node` do you have, running:
+
+```shell
+$ node -v
+v10.15.0
+```
+
+### Running
+
+In case you want to locally run the web interface from sources, you can choose one of the following:
+- using web assets from `bindata`, that will require you restart the server everytime you change any web asset.
```

Comment:
It's not yet merged, but this will not be `bindata` after #438.

Reply:
already merged; working on that :dagger: 
thanks!